axelor-supplychain/src/main/resources/views/AccountingBatch.xml

```diff
@@ -3,6 +3,32 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.4.xsd">
 
+  <form name="accounting-batch-form" id="supplychain-accounting-batch-form"
+    title="Accounting Batch" model="com.axelor.apps.account.db.AccountingBatch" extension="true">
+    <extend target="//field[@name='company']">
+      <attribute name="onChange"
+        value="action-accounting-batch-attrs-set-empty-year-and-accountset-value,action-method-accounting-batch-set-default-accounts"/>
+    </extend>
+    <extend target="//field[@name='miscOpeJournal']">
+      <insert position="before">
+        <field name="forecastedInvCustAccount" requiredIf="accountingCutOffTypeSelect &lt;= 2"
+          domain="self.statusSelect = 1 AND self.company = :company" grid-view="account-grid"
+          form-view="account-form" colSpan="6" showIf="accountingCutOffTypeSelect &lt;= 2"/>
+        <field name="forecastedInvSuppAccount" requiredIf="accountingCutOffTypeSelect &lt;= 2"
+          domain="self.statusSelect = 1 AND self.company = :company" grid-view="account-grid"
+          form-view="account-form" colSpan="6" showIf="accountingCutOffTypeSelect &lt;= 2"/>
```

Comment:
There is a `showIf` on the panel containing `miscOpeJournal` so maybe its condition should be added to the `requiredIf` to prevent any issue